src/migrations/1727167973931-migrate-guest-account.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1727167973931 implements MigrationInterface {
+  name = 'migrate-guest-account-1727167973931';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `
+    UPDATE account
+    SET 
+        type = 'guest', 
+        email = NULL, 
+        is_validated = false, 
+        item_login_schema_id = item_login_schema.id 
+    
+    FROM item_membership 
+        INNER JOIN item_login_schema 
+            ON item_login_schema.item_path = item_membership.item_path 
+    WHERE item_membership.account_id = account.id 
+    AND account.email ~* '^[0-9a-f]{4}-[0-9]{10,}@graasp.org' 
+    AND account.type = 'individual'
+    AND (
+      item_membership.account_id = item_membership.creator_id
+      OR item_membership.creator_id = '12345678-1234-1234-1234-123456789012'
```

Comment:
What is this creator id? Maybe add a comment to explain.

Reply:
At the very beginning of Graasp, public things were created by Graasper (this id). We don't use this fake member anymore.

Reply:
Thanks for this explanation